Sources/Private/CoreAnimation/Animations/CombinedShapeAnimation.swift

```diff
@@ -82,3 +82,10 @@ extension CombinedShapeItem {
       name: name)
   }
 }
+
+// MARK: Sendable
+
+/// `CombinedShapeItem` inherits `@unchecked Sendable` from `ShapeItem` and
+/// we need to restate that here to avoid a warning in Xcode 16
+// swiftlint:disable:next no_unchecked_sendable
+extension CombinedShapeItem: @unchecked Sendable { }
```

Comment:
If this works I would suggest putting the `Sendable` conformance in the main declaration instead of as an extension, like:

```swift
final class CombinedShapeItem: ShapeItem, Sendable {
```

Reply:
Unfortunately that still doesn't silence the warning

Reply:
Makes sense, thanks for checking